src/views/ApartmentApp/index.js

```diff
@@ -154,66 +165,81 @@ export default class ApartApp extends Component {
       const networkStatus = JSON.parse(localStorage.getItem('network-status')) || 'online';
 
       if (networkStatus === 'online' && this.state.isStu && this.props.Authentication) {
-        return (
-          <div className="apartment-application">
-            {this.state.loading && <GordonLoader />}
-            {!this.state.loading && (
-              <div className="apartment-application-card">
-                <Card>
-                  <CardContent
-                    style={{
-                      margin: 'auto',
-                      textAlign: 'center',
-                    }}
-                  >
-                    <h1>Apartment Application</h1>
-                    <br />
-                    <div className="applicant-menu-container">
-                      <div className="applicant-menu-column-1 box">
-                        <ApplicantList
-                          onApplicantRemove={this.onApplicantRemove}
-                          applicants={this.state.applicants}
-                          userProfile={this.state.userProfile}
-                        />
-                      </div>
-                      <div className="applicant-menu-column-2 box">
-                        <ApartmentPeopleSearch
-                          onSearchSelect={this.onSearchSubmit}
-                          Authentication={this.props.Authentication}
-                        />
-                      </div>
-                    </div>
-                    <br />
-                    <br />
-                    <TextField
-                      fullWidth
-                      value="Placeholder for Hall Preference Selection Menu"
-                      size="large"
-                      variant="outlined"
-                      InputProps={{
-                        readOnly: true,
-                      }}
-                    />
-                  </CardContent>
-                </Card>
-                <br />
-                <Card>
-                  <CardContent
-                    style={{
-                      margin: 'auto',
-                      textAlign: 'center',
-                    }}
-                  >
-                    <h2>Hello World:</h2>
-                    <h3>{'You name: ' + this.state.userProfile.fullName}</h3>
-                    <h3>{'On/Off Campus: ' + this.state.onOffCampus}</h3>
-                    <h3>{'Your room number: ' + this.state.onCampusRoom}</h3>
-                    <br />
-                  </CardContent>
-                </Card>
-              </div>
-            )}
-          </div>
+        return this.state.loading ? (
+          <GordonLoader />
+        ) : (
+          <Grid container direction="row-reverse" justify="center" spacing={2}>
+            <Grid item xs={12} md={4}>
+              <Card>
+                <CardContent
```

Comment:
Could this styling be done in a .scss file? It could then save the repetition on 194

Reply:
It could be done that way, but it would actually add significantly more lines of code because the `Grid` components are essentially just referencing a set a modular css definitions from `material-ui`.
While they may be confusing to work with at first, I have found that the `Grid` components make the styling a lot more consistent and reliable across various browsers, because it allows you to define the styling with those shorthand properties that would equivalently take maybe a dozen line in the .scss file (especially the rules for handling the breakpoints)

Reply:
Good point. What about the style for CardContent, though?

Reply:
That styling is actually kind-of redundant and unnecessary in this page, it was leftover from when we originally copied from timesheets. I have since removed that and also split up the cards to match the visual style things like the Involvements page or MyProfile (I just discovered the material-ui `CardHeader` components which the MyProfile page uses, and those also greatly simplify the code